Disable SOVERSION on a module. cmake>=3.4 does not honor it anymore, and this
breaks the PLIST depending on cmake version.

--- src/CMakeLists.txt~	2012-01-30 21:32:43.000000000 +0100
+++ src/CMakeLists.txt	2016-05-17 14:42:48.895675579 +0200
@@ -59,6 +59,5 @@
 ADD_DEPENDENCIES(roboptim-core-plugin-dummy roboptim-core)
 TARGET_LINK_LIBRARIES(roboptim-core-plugin-dummy roboptim-core)
 SET_TARGET_PROPERTIES(roboptim-core-plugin-dummy PROPERTIES
-  PREFIX ""
-  SOVERSION 1.1.0)
+  PREFIX "")
 INSTALL(TARGETS roboptim-core-plugin-dummy DESTINATION ${PLUGINDIR})
